go.mod

```diff
@@ -1,8 +1,25 @@
 module github.com/csconfederation/demoScrape2
 
-go 1.16
+go 1.21
+
+toolchain go1.21.3
 
 require (
-	github.com/markus-wa/demoinfocs-golang/v2 v2.10.0
+	github.com/markus-wa/demoinfocs-golang/v4 v4.0.0-beta.5.0.20231026145227-3c4b79fde38c
```

Comment:
looks like this can be bumped to `v4.0.0`

`github.com/markus-wa/demoinfocs-golang/v4 v4.0.0`

Reply:
Updated